GenderPayGap.WebUI/Views/Register/ReviewAddress.cshtml

```diff
@@ -89,7 +106,7 @@
                     </tr>

```

Comment:
A couple of lines above here is 
```cshtml
@await Html.PartialAsync("_SicCodeList", Model.SicCodes)
```

We should also convert `_SicCodeList.cshtml` to the new-style classes.
Also, take a look at how we display SIC codes in `ViewOrganisation.cshtml` (lines 188-197) - I think it looks a bit nicer
If we're converting `_SicCodeList.cshtml`, it would be nice to see if we could remove
```cshtml
@inject IDataRepository DataRepo;
```
It's a bit of a bad practice to do database requests from views